docs/src/components/SimpleCard/index.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from "./simple-card.module.css";
+
+export function SimpleGrid({ children }): JSX.Element {
+  console.log("styles", styles);
+  return <article className={styles.simpleGrid}>{children}</article>;
+}
+
+export function SimpleCard({ headerText, link, text }): JSX.Element {
```

Comment:
The style of this component does match the original docs, but looks kind of out of place in the docusaurus context. Could we make these look something like the cards in the [Stripe docs](https://docs.stripe.com/get-started)?

https://github.com/user-attachments/assets/2c8513fa-b0c3-4ce9-bbea-f3f80f87abf8

i.e. drop shadow only appears on hover, otherwise it's just a simple flat outline